src/System.Private.ServiceModel/tools/test/SelfHostWcfService/TestResources/WebSocketResources.cs

```diff
@@ -47,4 +42,54 @@ protected override Binding GetBinding()
             return binding;
         }
     }
+
```

Comment:
Nit: I kind of prefer a file per class, though I know we already violate that practice.  Better discovery.  Not a required change, just a suggestion if you refactor things.